core/org.wso2.carbon.bootstrap/src/test/java/org/wso2/carbon/bootstrap/logging/LoggingBridgeRegisterTest.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright 2017 WSO2, Inc. (http://wso2.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.wso2.carbon.bootstrap.logging;
+
+import org.apache.log4j.Appender;
+import org.apache.log4j.ConsoleAppender;
+import org.testng.annotations.Test;
+
+import static org.testng.Assert.assertNull;
+
+public class LoggingBridgeRegisterTest {
+
+    private static LoggingBridge loggingBridge;
+
+    /**
+     * Test if adding appender not implementing the LoggingBridge throws Exception
+     */
+    @Test(expectedExceptions = ClassCastException.class,
+            groups = {"org.wso2.carbon.utils.logging"},
+            description = "")
```

Comment:
Can move the method comment to the "description" in the annotation rather keeping that as empty.

Reply:
Fixed in 1fbd44f